packages/features/auth/signup/handlers/calcomHandler.ts

```diff
@@ -7,14 +7,14 @@ import { sendEmailVerification } from "@calcom/features/auth/lib/verifyEmail";
 import { WEBAPP_URL } from "@calcom/lib/constants";
 import { getLocaleFromRequest } from "@calcom/lib/getLocaleFromRequest";
 import { HttpError } from "@calcom/lib/http-error";
+import { usernameHandler, type RequestWithUsernameStatus } from "@calcom/lib/server/username";
```

Comment:
Use the re-usable file, not entirely sure which file is the right one but this was the one I touched.